gradio/components/json_component.py

```diff
@@ -83,9 +84,18 @@ def postprocess(self, value: dict | list | str | None) -> dict | list | None:
         if value is None:
             return None
         if isinstance(value, str):
-            return json.loads(value)
+            return orjson.loads(value)
```

Comment:
@abidlabs @freddyaboulton Apart from this PR,
I found this implementation is different from the doc string which says
> Expects a `str` filepath to a file containing valid JSON

while `json.loads()` expects a JSON string, not a file path (but `json.load()` does).

Which is the correct, the impl or the doc?

Reply:
Thanks, I updated the docstring about it together.

Reply:
Nice catch @whitphx